community/cypher/cypher-compiler-3.2/src/main/scala/org/neo4j/cypher/internal/compiler/v3_2/planner/execution/PipeExecutionPlanBuilder.scala

```diff
@@ -146,260 +149,264 @@ class PipeExecutionPlanBuilder(clock: Clock, monitors: Monitors, pipeBuilderFact
+      case RelationshipCountFromCountStore(IdName(ident), startLabel, typeNames, endLabel, _) =>
+        RelationshipCountFromCountStorePipe(ident, startLabel.map(LazyLabel.apply), typeNames, endLabel.map(LazyLabel.apply))(id = id)
 
-    case NodeByLabelScan(IdName(id), label, _) =>
-      NodeByLabelScanPipe(id, LazyLabel(label))()
+      case NodeByLabelScan(IdName(ident), label, _) =>
+        NodeByLabelScanPipe(ident, LazyLabel(label))(id = id)
 
-    case NodeByIdSeek(IdName(id), nodeIdExpr, _) =>
-      NodeByIdSeekPipe(id, nodeIdExpr.asCommandSeekArgs)()
+      case NodeByIdSeek(IdName(ident), nodeIdExpr, _) =>
+        NodeByIdSeekPipe(ident, nodeIdExpr.asCommandSeekArgs)(id = id)
 
-    case DirectedRelationshipByIdSeek(IdName(id), relIdExpr, IdName(fromNode), IdName(toNode), _) =>
-      DirectedRelationshipByIdSeekPipe(id, relIdExpr.asCommandSeekArgs, toNode, fromNode)()
+      case DirectedRelationshipByIdSeek(IdName(ident), relIdExpr, IdName(fromNode), IdName(toNode), _) =>
+        DirectedRelationshipByIdSeekPipe(ident, relIdExpr.asCommandSeekArgs, toNode, fromNode)(id = id)
 
-    case UndirectedRelationshipByIdSeek(IdName(id), relIdExpr, IdName(fromNode), IdName(toNode), _) =>
-      UndirectedRelationshipByIdSeekPipe(id, relIdExpr.asCommandSeekArgs, toNode, fromNode)()
+      case UndirectedRelationshipByIdSeek(IdName(ident), relIdExpr, IdName(fromNode), IdName(toNode), _) =>
+        UndirectedRelationshipByIdSeekPipe(ident, relIdExpr.asCommandSeekArgs, toNode, fromNode)(id = id)
 
-    case NodeIndexSeek(IdName(id), label, propertyKey, valueExpr, _) =>
-      val indexSeekMode = IndexSeekModeFactory(unique = false, readOnly = readOnly).fromQueryExpression(valueExpr)
-      NodeIndexSeekPipe(id, label, propertyKey, valueExpr.map(buildExpression), indexSeekMode)()
+      case NodeIndexSeek(IdName(ident), label, propertyKey, valueExpr, _) =>
+        val indexSeekMode = IndexSeekModeFactory(unique = false, readOnly = readOnly).fromQueryExpression(valueExpr)
+        NodeIndexSeekPipe(ident, label, propertyKey, valueExpr.map(buildExpression), indexSeekMode)(id = id)
 
-    case NodeUniqueIndexSeek(IdName(id), label, propertyKey, valueExpr, _) =>
-      val indexSeekMode = IndexSeekModeFactory(unique = true, readOnly = readOnly).fromQueryExpression(valueExpr)
-      NodeIndexSeekPipe(id, label, propertyKey, valueExpr.map(buildExpression), indexSeekMode)()
+      case NodeUniqueIndexSeek(IdName(ident), label, propertyKey, valueExpr, _) =>
+        val indexSeekMode = IndexSeekModeFactory(unique = true, readOnly = readOnly).fromQueryExpression(valueExpr)
+        NodeIndexSeekPipe(ident, label, propertyKey, valueExpr.map(buildExpression), indexSeekMode)(id = id)
 
-    case NodeIndexScan(IdName(id), label, propertyKey, _) =>
-      NodeIndexScanPipe(id, label, propertyKey)()
+      case NodeIndexScan(IdName(ident), label, propertyKey, _) =>
+        NodeIndexScanPipe(ident, label, propertyKey)(id = id)
```

Comment:
There is already an `id` in scope, which clashes with some of the these `case`s, which is why I renamed them to `ident`